app/components/ui/molecules/BurgerMenu.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Box } from 'grid-styled'
+import { th } from '@pubsweet/ui-toolkit'
+import Icon from '../atoms/Icon'
+import ButtonAsIconWrapper from '../atoms/ButtonAsIconWrapper'
+import CrossIconButton from '../atoms/CrossIconButton'
+import NavLink from '../../ui/atoms/NavLink'
+import ModalOverlay from '../molecules/ModalOverlay'
+
+const MenuPanel = styled.div`
+    max-width: 240px;
+    background-color: ${th('colorBackground')}
+    height: 100%;
+`
+
+const MenuItem = styled(Box)`
+  border-top: ${th('borderWidth')} ${th('borderStyle')} ${th('colorBorder')};
+  height: ${th('space.6')};
```

Comment:
I'm not sure whether this is necessary? The height seems to be generated well by the combination of line height on the navlink & padding that you've added to each `MenuItem`? But maybe I've missed something, and it's adding something more?

Reply:
This is a fix for the Cross MenuItem, It was setting its inner height to be 31px rather than 24px (standard line-height) of the other link items, Couldn't figure out why the Icon was doing this as its dimensions were 24 x 24 and line-height was the same as all others